interfaces/interface_EC-Earth4.yml

```diff
@@ -0,0 +1,192 @@
+# Configuration yaml file for EC-Earth4 model and variables
+# 
+# Possible filetype attributes for variables are defined in the model section
+# The full path is $basedir/$dir/$filename
+# 
+# basedir can be formatted with: expname
+# dir can be formatted with: expname
+# filename can be formatted with: expname, year1, year2
+#
+# The key 'component' is used to determine the specific type of grid fixes to apply
+# (defined in cdopipe for now)
+# 
+# We also have the concept of 'domain' (atm or oce) later used to choose which fix to apply
+# Each component belongs to one and only one domain
+
+# Definition of model output properties
+model:
+  basedir: '{expname}'
+  filetype: 
+    atm2d:
+      filename: '{expname}_atm_cmip6_1m_{year1}-{year2}.nc'
+      dir: output/oifs
+      component: oifs
+    atm3d:
+      filename: '{expname}_atm_cmip6_pl_1m_{year1}-{year2}.nc'
+      dir: output/oifs
+      component: oifs
+    oceT:
+      filename: '{expname}_oce_1m_T_{year1}-{year2}.nc'
+      dir: output/nemo
+      component: nemo
+    ice:
+      filename: '{expname}_ice_1m_{year1}-{year2}.nc'
+      dir: output/nemo
+      component: nemo
+
+  # This specifies what is the domain for each component (oce or atm)
+  # This is needed to select the correct fix in ecmean.py
+  # Specify also inifiles for components (an extra help file),
+  # containing extra info such as areas, used in set_gridfixes
+  # inifile can be formatted with: expname
+  # If a relative path is used this is attached to $ECEDIR/$basedir
+  component:
+    oifs:
+      domain: atm 
+      inifile: 'ICMGG{expname}INIT'
+    nemo:
+      domain: oce 
+      inifile: domain_cfg.nc
+
+# Definition of individual variables
```

Comment:
I would add a "variable" key and put the other below, just for clarity

Reply:
I am not sure that I understood this, could you make an example? Is it still valid now that I inverted the domain/component logic?

Reply:
sorry, I meant a trivial thing...

```
# Definition of individual variables
variables: 
    tas:
      -bla bla
    pr:
     - bla bla 

```

Reply:
Ah ok, I just implemented it.